### ISSUE TITLE
PHP warnings on CRON with 1.0.1.dev202310231802

### ISSUE THREAD

**Issue by sc00zy** [akeeba/panopticon#173]
I get PHP warnings since I installed the dev version 1.0.1.dev202310231802

PHP verouderd: het aanroepen van Awf\Application\Application::getInstance is verouderd. Ga in plaats daarvan door de container. [panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Application/Application.php](http://panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Application/Application.php) /var/www/vhosts/panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Application/Application.php online 133
PHP verouderd: het aanroepen van Awf\Application\Application::getInstance is verouderd. Ga in plaats daarvan door de container. in [panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Application/Application.php](http://panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Application/Application.php) online 133
PHP verouderd: het aanroepen van Awf\Mvc\Model::getTmpInstance is verouderd. Gebruik in plaats daarvan de MVCFactory-service van de container. in [panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Mvc/Model.php](http://panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Mvc/Model.php) online 137
PHP verouderd: het aanroepen van Awf\Mvc\Model::getTmpInstance is verouderd. Gebruik in plaats daarvan de MVCFactory-service van de container. in [panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Mvc/Model.php](http://panopticon.domainname.nl/httpdocs/vendor/akeeba/awf/src/Mvc/Model.php) online 137

And

PHP-waarschuwing: Ongedefinieerde [panopticon.welldotcom.nl/httpdocs/src/Task/RefreshInstalledExtensions.php](http://panopticon.domainname.nl/httpdocs/src/Task/RefreshInstalledExtensions.php) : stdClass::$locked in /var/www/vhosts/panopticon.domainname.nl/httpdocs/src/Task/RefreshInstalledExtensions.php online 365

The install is on PHP 8.1.24

Thanks

**Comment by nikosdion** [comment]
It did not start with 1.0.1, it has been there. You only noticed now because I had you turn on Debug when troubleshooting your connector issue. If you turn off Debug they will disappear.

The first ones come from the framework itself, as I've recently refactored it to make it stricter. I know about these and will fix them, I just needed to get 1.0.1 out because it fixed pretty high priority issues.

The latter is a PHP 8 deprecation notice. I am also aware of it. Same as above, I had to get 1.0.1 out.
